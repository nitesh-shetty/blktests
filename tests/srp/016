#!/bin/bash
# SPDX-License-Identifier: GPL-2.0+
# Copyright 2022 Google LLC

. tests/srp/rc

DESCRIPTION="RDMA hot-unplug"
QUICK=1

test_hot_unplug() {
	use_blk_mq y y || return $?
	# This triggers a use-after-free with older kernels. See also
	# https://lore.kernel.org/linux-rdma/17649b9c-7e42-1625-8bc9-8ad333ab771c@fujitsu.com/
	stop_soft_rdma
}

test() {
	trap 'trap "" EXIT; teardown' EXIT
	setup && test_hot_unplug && echo Passed
}
