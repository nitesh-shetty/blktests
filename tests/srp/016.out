Configured SRP target driver
Passed
